test/web-platform-tests/run-single-wpt.js

```diff
@@ -118,9 +124,16 @@ function createJSDOM(urlPrefix, testPath) {
           errors.push(...doneErrors);
           errors.push(...unhandledExceptions);
 
-          if (errors.length === 1) {
+          if (errors.length === 0 && expectFail === true) {
```

Comment:
Just as I request a review, I realise that I ought to simplify these to `expectFail` and `!expectFail`. Will do tomorrow.